### ISSUE TITLE
MultiHost and Firewall

### ISSUE THREAD

**Issue by IzzySoft** [mikaku/Monitorix#129]
Never having used this, I always thought the multi-host enabled server would be used to retrieve the pages from other hosts in this context. Now having a use-case for this (multiple servers behind a firewall, with just one exposed to HTTP/s) I've learned better.

Maybe a switch could be introduced controlling whether the client shall contact all hosts directly, or the "master" shall play proxy? Alternatively, I'd also accept a new MultiHostProxy group of course :)

**Comment by mikaku** [comment]
Yes, I know, this is a long standing pending feature.
I'll use that issue to send news on the progress.
Thanks.

**Comment by IzzySoft** [comment]
My pleasure. Big thanks from my end follows later then :)

Btw: Maybe that could be realized via an additional "flag" to the host as well (so we don't need another group). If it's possible to add that to the end, and have it default to "noproxy" when not set, it wouldn't break existing configs.

**Comment by sorenwacker** [comment]
I think, that is what I am facing right now. Connected via SSH tunnel to a main monitor node that should connect to all workers, but unable to see them, probably because the notebook from which I connect is in a different network.

**Comment by mikaku** [comment]
@sorenwacker, yes the browser in your notebook is the one who tries to fetch all the graphs from remote Monitorix.

As a work-around I'd recommend you to install a simple proxy (like [Tinyproxy](https://github.com/tinyproxy/tinyproxy)) in your "main monitor", that's how I do it in certain configurations.
